src/test/java/hw4/pages/components/RightSideBar.java

```diff
@@ -0,0 +1,44 @@
+package hw4.pages.components;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class RightSideBar {
+    private WebDriver driver;
+
+    @FindBy(name = "log-sidebar")
+    private WebElement logSidebar;
+
+    @FindBy(css = "[class='panel-body-list logs'] li")
```

Comment:
I think this can be simplified `.panel-body-list.logs li`

Reply:
fixed